packages/runtime/src/module-system.ts

```diff
@@ -184,6 +198,19 @@ export class ModuleSystem {
     }
   }
 
+  getModuleUrl(moduleId: string): string {
+    for (const [resource, modules] of this.resources) {
+      if (modules.includes(moduleId)) {
+        const publicPath = this.publicPaths[0];
+        // @ts-ignore
+        const url = `${window.FARM_HOST}:${window.FARM_PORT}${
```

Comment:
refer to the previous

Reply:
like const url = `${location.host}${publicPath === '/' ? '' : publicPath}/${resource}`;  ？

Reply:
can do this for a while, because it's uncertain.